Stock_Alert/main.py

```diff
@@ -18,40 +19,49 @@
     "apikey": stock_api_key
 }
 
-stock_response = requests.get(STOCK_ENDPOINT, params=stock_params)
-stock_data = stock_response.json()["Time Series (Daily)"]
+stock_response = Request(STOCK_ENDPOINT).make(stock_params)
```

Comment:
```suggestion
stock_response = Request(STOCK_ENDPOINT).make(stock_params).json()
```
just so that you don't need to convert each time